### PR TITLE
[GR] bump version

### PR DESCRIPTION
**Pull request by jheinen**
bump GR version to 0.72.5

### CODE REVIEW

**Comment by jheinen** [comment]
The `x86-64-apple-darwin` build fails because `macosx-version-min` is set to `10.10` (see compiler flags). How can I set them to `10.14`?

**Comment by giordano** [comment]
Try setting
```sh
export MACOSX_DEPLOYMENT_TARGET=10.14
```
for `x86_64-apple-darwin`, if that's not enough because you actually need a 10.14 sdk then you have to use that as a source
https://github.com/JuliaPackaging/Yggdrasil/blob/806bbbf2cc4882aa3dc3de98fc2924953b1e72f2/O/OpenSpiel/build_tarballs.jl#L23
https://github.com/JuliaPackaging/Yggdrasil/blob/806bbbf2cc4882aa3dc3de98fc2924953b1e72f2/O/OpenSpiel/build_tarballs.jl#L28-L36

**Comment by jheinen** [comment]
@giordano : Thanks for your help!

File: G/GR/build_tarballs.jl [hunk 1]
```diff
@@ -31,7 +33,17 @@ else
...
+    pushd $WORKSPACE/srcdir/MacOSX10.*.sdk
+    rm -rf /opt/${target}/${target}/sys-root/System
+    cp -ra usr/* "/opt/${target}/${target}/sys-root/usr/."
+    cp -ra System "/opt/${target}/${target}/sys-root/."
+    export MACOSX_DEPLOYMENT_TARGET=10.14
+    popd
+fi
+
 if [[ "${target}" == *apple* ]]; then
+    export MACOSX_DEPLOYMENT_TARGET=10.14
```
Comment (giordano) [inline G/GR/build_tarballs.jl:46]: This is done already 5 lines above and is necessary only on x86_64 anyway 
```suggestion
```

**Pull request merged**